inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sink/mysql/MySQLSinkDTO.java

```diff
@@ -218,7 +218,7 @@ public static String setDbNameToUrl(String jdbcUrl, String databaseName) {
      * @return str without sensitive param
      */
     @VisibleForTesting
```

Comment:
This method was called by non-testing classes, so please remove this annotation.